acceptance-tests/src/test/java/tech/pegasys/signers/dsl/hashicorp/HashicorpVaultDocker.java

```diff
@@ -60,11 +62,14 @@
   private static final String DEFAULT_VAULT_HOST = "localhost";
   private static final int DEFAULT_VAULT_PORT = 8200;
 
-  private static final String SECRET_PATH = "/secret";
+  private static final String VAULT_ROOT_PATH = "/secret";
   private static final String SIGNING_KEY_RESOURCE = "/signingKeyResource";
-  private static final String VAULT_PUT_RESOURCE = SECRET_PATH + SIGNING_KEY_RESOURCE;
+  private static final String VAULT_PUT_RESOURCE = VAULT_ROOT_PATH + SIGNING_KEY_RESOURCE;
   private static final String VAULT_SIGNING_KEY_PATH =
-      "/v1" + SECRET_PATH + "/data" + SIGNING_KEY_RESOURCE;
+      "/v1" + VAULT_ROOT_PATH + "/data" + SIGNING_KEY_RESOURCE;
```

Comment:
should we ask user complete url path? i.e. `/v1/secret/data/MySecretKey`?

Reply:
instead of assuming that hashicorp will always run on `/v1` and that the admin has enabled `/secret`?

Reply:
Our production code assumes the user will input the FULL Http path to the key, rather than just "mysecretkey". 
Otherwise there are simply too many things to insert to the commandline eg:
* Vault Root Path
* Http API version
All of that information is effectively encoded in the key path.

Reply:
👍